CSV/brands.csv

```diff
@@ -115,7 +116,17 @@ M4P$20,MPEG-4 protected audio,iTunes
 M4V$20,MPEG-4 protected audio+video,iTunes
 MFSM,Media File for Samsung video Metadata,Samsung
 MGSV,Home and Mobile Multimedia Platform (HMMP),Sony
+MiAB,Multi-Image Application format brand,MIAF
```

Comment:
Can we use more specific descriptions for the brands?

Reply:
don't forget to remove iso8 from brands.csv also...

Reply:
(and the space after the comma would also be an error)